promtsdb/promtsdb_integration_test.go

```diff
@@ -28,7 +28,7 @@ func TestTsdbIntegration(t *testing.T) {
 	}
 	fmt.Println(cfg)
 
-	adapter, err := NewV3ioProm(cfg, nil, nil)
+	adapter, err := NewV3ioProm(cfg, nil, nil, false)
```

Comment:
Just leave it out of the constructor? Has the added benefit of making it backward compatible.